application/src/main/java/com/sanction/thunder/resources/VerificationResource.java

```diff
@@ -39,13 +46,100 @@
    * @param metrics The metrics object to set up meters with.
    */
   @Inject
-  public VerificationResource(PilotUsersDao usersDao, MetricRegistry metrics) {
+  public VerificationResource(PilotUsersDao usersDao,
+                              MetricRegistry metrics,
+                              EmailService emailService) {
     this.usersDao = usersDao;
+    this.emailService = emailService;
 
     // Set up metrics
-    this.postRequests = metrics.meter(MetricRegistry.name(
+    this.getRequests = metrics.meter(MetricRegistry.name(
         UserResource.class,
-        "post-requests"));
+        "get-requests"));
```

Comment:
Make sure you rename it here in the string too. Also, change the `UserResource.class` to `VerificationResource.class`, I didn't catch that before.